src/main/java/com/couchbase/lite/router/Router.java

```diff
@@ -1563,6 +1564,7 @@ public Status do_GET_Document_changes(Database _db, String docID, String _attach
             Log.v(Log.TAG_ROUTER, "Filter params=" + changesFilterParams);
         }
 
+        // changesSince() is query only. not required synchronized
         RevisionList changes = db.changesSince(since, options, changesFilter, changesFilterParams);
```

Comment:
This comment might not be correct if want to make calling to the database object thread-safe.